plugins/versionlock.py

```diff
@@ -152,6 +154,11 @@ def run(self):
                 self.opts.package.insert(0, self.opts.subcommand)
             elif self.opts.subcommand in EXC_CMDS:
                 cmd = 'exclude'
+            elif self.opts.subcommand in DEP_EXC_CMDS:
+                msg = "Subcommand '{}' is deprecated. Use 'exclude' subcommand instead.".format(
+                    self.opts.subcommand)
+                warnings.warn(msg, DeprecationWarning, stacklevel=2)
```

Comment:
We have dnf specific subclass of `DeprecationWarning` in `dnf.exceptions`. Maybe we should use it?

Reply:
I am really not sure. I tried following but it did not provide any message to the terminal.
```
dnf.exceptions.DeprecationWarning(msg)
#warnings.warn(msg, DeprecationWarning, stacklevel=2)
```
What I really don't want is to raise an exception because then it will change the behaviour. @m-blaha What do you think?

Reply:
Finally I get how to change it. I am sorry for delay.

Reply:
Yes, this is what I meant by my comment :)